specification/trace/semantic_conventions/messaging.md

```diff
@@ -80,6 +81,45 @@ Often such destinations are unnamed or have an auto-generated name.
 
 Given these definitions, the remainder of this section describes the semantic conventions for Spans describing interactions with messaging systems.
 
+### Context propagation
+
+A message may traverse many different components and layers in one or more intermediaries
+when it is propagated from the producer to the consumer(s). To be able to correlate
+consumer traces with producer traces using the existing context propagation mechanisms,
+all components must propagate context down the chain.
+
+This imposes a requirement that all components are instrumented
+and propagate context according to OpenTelemetry requirements. In reality,
+this cannot be assumed, and in many cases is not even desired or possible.
```

Comment:
```suggestion
this cannot be assumed, and in many cases is not desired or possible.
```

Reply:
Removed this sentence so not necessary anymore [16bbdc7](https://github.com/dynatrace-oss-contrib/opentelemetry-specification/pull/22/commits/16bbdc7d4447813da7e3209cd47437700fcd4a65)